R-package/vignettes/classifyRealImageWithPretrainedModel.Rmd

```diff
@@ -40,7 +41,7 @@ model = mx.model.load("Inception/Inception_BN", iteration=39)
 We also need to load in the mean image, which is used for preprocessing using ```mx.nd.load```.
 
 ```{r}
-mean.img = as.array(mx.nd.load("Inception/mean_224.nd")[["mean_img"]])
```

Comment:
I think I know what you mean. Let's make this clear: `mx.nd.load` should always return a list.

Right now, when loading a list of only element, it will return the element.

https://github.com/dmlc/mxnet/blob/master/R-package/R/ndarray.R#L16-L20

Let's make sure on this point, so we don't have misunderstanding in the future. @hetong007